db/fixtures/tactical_vote_stt_csv.rb

```diff
@@ -17,7 +17,7 @@ class TacticalVoteCsv
   def initialize
     @file_name = FILE_NAME
     @link = "https://stopthetories.vote/"
-    @site = "StopTheTories"
+    @site = "stop-the-tories"
```

Comment:
If this is what gets displayed in the boxes, TBH this looks a bit ugly to me, but we can debate that later ;-)

Reply:
The reason for the hyphens, I discovered, is that if you have 7-8 recommendations (which we DID have) the hyphens nicely wrap on the horizontal version of the layout. 

But yeah, I'm sure a proper web designer could do a better job.

Reply:
I believe the correct solution to that is [a zero width space](https://stackoverflow.com/a/15841796/179332).